pootle/apps/pootle_store/form_fields.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django import forms
+from django.utils.datastructures import MergeDict, MultiValueDict
+
+from pootle.core.dateparse import parse_datetime
+from pootle_misc.checks import CATEGORY_IDS
+
+
+class CommaSeparatedCheckboxSelectMultiple(forms.CheckboxSelectMultiple):
+
+    def value_from_datadict(self, data, files, name):
+        # Accept `sfields` to be a comma-separated string of fields (#46)
+        if "," in data.get(name, ""):
+            return data.get(name).split(u',')
+        if isinstance(data, (MultiValueDict, MergeDict)):
+            return data.getlist(name)
+        return data.get(name, None)
+
+
+class MultipleValueWidget(forms.TextInput):
+
+    def value_from_datadict(self, data, files, name):
+        if hasattr(data, "getlist"):
+            return data.getlist(name)
+        return []
+
+
+class MultipleArgsField(forms.Field):
+    widget = MultipleValueWidget
+
+    def __init__(self, *la, **kwa):
```

Comment:
2 reasons:
- pdb has ``args`` so it conflicts with locals() which is a massive pita!
- kwargs has 3 extra chars that i hate having to type like all the time

that said - i try and clean this up before submitting - so apologies - ill fix now